### ISSUE TITLE
Poor performance with time_inceptionV3_1.5987_loss.h5

### ISSUE THREAD

**Issue by bigzhao** [yashk2810/Image-Captioning#5]
When i try your weights name 'time_inceptionV3_1.5987_loss.h5', the performance of image-caption model is poor.

Taking the basketball image as a example, the output is as follows:
"""Normal Max search: man ."""

Have you met this problem before?

PS: My keras version is 2.1.2. 

============================
I also load this weights on Keras 1.2.2 and Tensorflow 0.12.1, but it failed again. It seems the weights file 'time_inceptionV3_1.5987_loss.h5' is useless.

**Comment by yashk2810** [comment]
Yes, I am also facing this problem. You will have to retrain the network from scratch. I would suggest using Google Colab(if you don't have your own GPU) notebooks as they give free GPU compute time

**Comment by SwetaKaman** [comment]
But when I'm using ('time_inceptionV3_3.15_loss.h5') its showing almost accurate captions

**Comment by gitlost-murali** [comment]
@yashk2810 & @bigzhao  I'm facing the same problem too. Are you getting better outputs after training from scratch again ?

**Comment by SwetaKaman** [comment]
Yes I'm getting good output with this weight file. Not upto the mark but
yes I'm getting good output. May be you just try once to save as well as
load the weight file(1.5987) after the last epoch of the training.

**Comment by FranciscoGomez90** [comment]
Hi there! Can you share the time_inceptionV3_2.8876_loss.h5 file? It seems that the DL link from the repository is outdated =(
